nav2_waypoint_follower/README.md

```diff
@@ -2,7 +2,8 @@
 
 The navigation2 waypoint follower is an example application of how to use the navigation action to complete some sort of orchestrated task. In this example, that task is to take a given set of waypoints and navigate to a set of positions in the order provided in the action request. The last waypoint in the waypoint array is the final position.
 
-The package exposes the `FollowWaypoints` action server of type `nav2_msgs/FollowWaypoints`. It is given an array of waypoints to visit, gives feedback about the current index of waypoint it is processing, and returns a list of waypoints it was unable to complete.
+The package exposes the `follow_waypoints` action server of type `nav2_msgs/follow_waypoints`.
```

Comment:
I think this sentence should be 
The package exposes the `follow_waypoints` action server of type `nav2_msgs/FollowWaypoints`.